### PR TITLE
chore: get rid of glob dependencies by switching del with rimraf

### PR DESCRIPTION
**Pull request by ludofischer**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [x] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### Motivation / Use-Case

Replacing del gets rid of about 10 extra dependencies,
most are related to globbing, like globby, fast-glob and dir-glob, but also others like fastq, queue-microtask, reusify.

### Breaking Changes

No breaking changes.

### Additional Info

del wraps rimraf in promisify and adds
1. more sophisticated globbing
2. deletion of multiple files at once by passing an array
3. a check to prevent deleting the current working directory and outside

Since the code base does not use any of the extra features, we can just wrap `rimraf` with `promisify()`.

### CODE REVIEW

File: lib/Server.js [hunk 1]
```diff
@@ -1133,13 +1133,15 @@ class Server {
 
           // cert is more than 30 days old, kill it with fire
           if ((now - Number(certificateStat.ctime)) / certificateTtl > 30) {
```
Comment (ludofischer) [inline lib/Server.js:1135]: This is the only production file that uses `del`, the rest of the usages are all in tests.

File: lib/Server.js [hunk 2]
```diff
@@ -1133,13 +1133,15 @@ class Server {
...
-            const del = require("del");
+            const { promisify } = require("util");
+            const rimraf = require("rimraf");
+            const del = promisify(rimraf);
 
             this.logger.info(
               "SSL certificate is more than 30 days old. Removing..."
             );
 
-            await del([certificatePath], { force: true });
```
Comment (ludofischer) [inline lib/Server.js:1142]: `certificatePath` is not a glob, it's the result of calling `const certificatePath = path.join(certificateDir, "server.pem")` on line 1119. 
Also line 1123 uses `fs.stat` on `certificatePath` to check for existence. 
`{force: true}` bypasses a check added by `del`, so it's not needed for rimraf.

**Review by ludofischer — COMMENTED** [review]
You can see how `del` wraps rimraf in the [del source](https://github.com/sindresorhus/del/blob/7c756b393804ad7df5cb534cecb80677565add59/index.js#L69).

**Review by alexander-akait — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]
LGTM. Thank you.

**Comment by alexander-akait** [comment]
@snitin315 Feel free to merge when CI will be green

**Pull request merged**